nmostesting/suites/IS1101Test.py

```diff
@@ -1733,29 +1744,31 @@ def test_04_03(self, test):
             """
             The test requires streaming from a Sender in order to
             verify the state of the Receiver and the associated outputs.
-            """
-            """
-            if IS11_RefecenceSender is not configured fail 4.3 with log indicating
-            that IS11_RefecenceSender is not configured
+            if IS11_REFERENCE_SENDER is not configured fail 4.3 with log indicating
+            that IS11_REFERENCE_SENDER is not configured
             """
             if len(CONFIG.IS11_REFERENCE_SENDER) == 0:
```

Comment:
Idiomatically, testing a variable that should be a non-zero-length string, probably `if not` rather than `if len(...) == 0:`, to account for empty string, `None`, etc.

Reply:
Applied suggestion